docs/API_v1/transactions.md

```diff
@@ -6,8 +6,40 @@ Transactions API
 
 
 ### Payment transaction  [POST] 
-//TODO: How to make a transaction and sign 
 
++ Data Body consist of 3 parts, ; T, H, B
+    + T : ‘transaction’
+
+    + H : H means Header. it consists of version, hash, signature & created.
```

Comment:
Stylistic comment: "X means Y" is repetitive and not engaging. I usually recommend a shorter style, e.g.:
```
H: Header, consisting of `Version`, `Hash` and `Signature`
 - Version: Currently fixed to 1
 - Hash: sha256(sha256(B)) <== With a link to `B` section here
 - Signature: xdr.Sign(ed25519.Sign(Hash, NetworkID, Seed))
    We use [Stellar's key](https://github.com/stellar/go/tree/master/keypair) for signing.
    We also provide a[JavaScript](https://github.com/bosnet/sebakjs-util) and [Python](https://github.com/spikeekips/sebakpy-util) SDK.

Reply:
Oh thanks @Geod24 .
I like your style. 
I will try it. 

but you should generate API doc through this command `https://github.com/bosnet/sebak/tree/master/docs/API_v1`